shared/src/business/utilities/DateHandler.ts

```diff
@@ -2,6 +2,7 @@ import { DateTime, DurationLike, Interval } from 'luxon';
 import fedHolidays from '@18f/us-federal-holidays';
 
 export const FORMATS = {
+  CURRENT_AS_OF_TIMESTAMP: "MM/dd/yy h:mm a 'Eastern'",
```

Comment:
10460 handled similar logic differently. (I use a helper that returns, among other things, a formatNow according to this new format. 10460 used an action with global state.) Once this feature is delivered, I will discuss with the 10460 crew and open a follow-up PR to clean up a few things to make work more consistent.